tests/trainers/test_byol.py

```diff
@@ -76,7 +74,6 @@ def test_trainer(
 
         # Instantiate model
         model = instantiate(conf.module)
-        model.backbone = SegmentationTestModel(**conf.module)
```

Comment:
`BYOLTask` has never had a `backbone` attribute. This line of code didn't do anything, and the tests are still slower and require more memory than necessary.